helm-chart/kuberay-operator/values.yaml

```diff
@@ -55,3 +55,12 @@ readinessProbe:
 
 createCustomResource: true
 rbacEnable: true
+
+# Respect the "restricted" security policy
+securityContext:
+  allowPrivilegeEscalation: false
+  capabilities:
+    drop: ["ALL"]
+  runAsNonRoot: true
+  seccompProfile:
+    type: RuntimeDefault
```

Comment:
Let's actually comment it out to leave the default configuration as is.
Then add a comment that the commented out configuration is recommended for secure production deployments.

Reply:
Removed the default `securityContext` in `values.yaml`, and updated `pod-security.md`. https://github.com/ray-project/kuberay/pull/750/commits/b6e1465059b7c3ffe2b9ee03b99f98946d722501